### ISSUE TITLE
Possible for subgroups aswell?

### ISSUE THREAD

**Issue by Slluxx** [Dorukyum/pycord-multicog#3]
EG:

```py
admin = discord.SlashCommandGroup("admin", "Utility commands")
admin_faq = admin.create_subgroup("admin_faq", "FAQ admin commands")

# different file

@add_to_group("admin_faq")
# or @add_to_group("admin.admin_faq")
```

**Comment by Dorukyum** [comment]
Multicog only works for top-level groups at the moment but I think I can add support for subgroups as well. Why don't you [join the support server](https://discord.gg/8JsMVhBP4W) to be notified?

**Comment by Dorukyum** [comment]
This has now been implemented and a new release (v1.1.0) was published, thanks!
